app/auth/views/fido.py

```diff
@@ -0,0 +1,106 @@
+import json
+import secrets
+import webauthn
+from app.config import URL as SITE_URL
```

Comment:
same comment as in `fido_setup`, any reason to change the import name here?

Reply:
No longer valid due to https://github.com/simple-login/app/pull/159#discussion_r421343108